packages/api/test/addbuild-getbuild.js

```diff
@@ -341,8 +348,8 @@ describe('Add-Build', () => {
 
       assert(ansObj[2].name === 'a/4'); // failed on build before the last build
 
-      assert(ansObj[3].name === 'a/3' || ansObj[3].name === 'a/1'); // both failed on most recent build
-      assert(ansObj[4].name === 'a/3' || ansObj[4].name === 'a/1'); // both failed on most recent build
+      assert(ansObj[3].name === 'a/3'); // passed on build before the last build
+      assert(ansObj[4].name === 'a/1'); // has not been recently updated because considered healthy
```

Comment:
the a/1 test has been considered healthy from the first build, so it will appear last in the list